lib/lumberjack/server.rb

```diff
@@ -253,8 +266,8 @@ def read_socket(&block)
           # We receive a new payload
           window_size(*args)
           reset_next_ack
-        when :data
-          data(*args, &block)
+        when :data, :json
+          data(event, *args, &block)
```

Comment:
In this part of the code we are using the variable `event` in the `#data` method we use code, might be good to use the same name?